vagrant/ansible/files/config/default.properties

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2017 Saxon State and University Library Dresden (SLUB)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# qucosa fedora commons 3 reporting db configuration
+db.url=jdbc:postgresql://localhost:5432/reporting
+db.driver=org.postgresql.Driver
+db.user=reportingDB
+db.passwd=reportingDB
+oai.url=http://sdvcmr-prod-core01.slub-dresden.de:8080/fedora/oai
```

Comment:
Although the system host will not work from outside, those hostnames should never appear in public repositories.
Consider using default hosts like `localhost` and replace it via external configuration at ansible run.

See http://docs.ansible.com/ansible/latest/playbooks_variables.html#variable-file-separation:

> It’s a great idea to keep your playbooks under source control, but you may wish to make the playbook source public while keeping certain important variables private. Similarly, sometimes you may just want to keep certain information in different files, away from the main playbook.